src/wp-includes/kses.php

```diff
@@ -2282,6 +2286,7 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 			'column-width',
 
 			'color',
+			'filter',
```

Comment:
@noisysocks is already including most of these changes in https://github.com/WordPress/wordpress-develop/pull/1804. Excluding filter, because filter was added in a different way than the others (in lib/global-styles.php) instead of lib/compat.php. Should we just update the plugin to add this in lib/compat.php and update Robert PR to contain all kses changes and remove kses from here?

Reply:
Unfortunately, we need the kses changes in this PR so the code works. Perhaps if this lands quicker than #1804, those changes can be removed from there.

Reply:
https://github.com/WordPress/wordpress-develop/pull/1804 will land last. If there is overlap then it will come up when I rebase my PR, so no worries 🙂